### PR TITLE
Students and Graduates page

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Adding the students and graduates page

## Additions

- Added `careers/students-and-graduates` page
- Added LESS file for careers pages

## Testing

- Visit `careers/students-and-graduates`

## Review

- @anselmbradford 
- @jimmynotjim 
- @sebworks 
- @sarahsimpson09 (for copy!)

## Preview

image

[Preview this PR without the whitespace changes](?w=0)

### CODE REVIEW

**Comment by sebworks** [comment]
:+1:

File: careers/_vars-careers.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% set path = '/careers/' %}
+{% set nav_items = [
+    (path, 'index', 'Careers at the CFPB'),
+    (path + 'working-at-cfpb/', 'working-at-cfpb', 'Working at CFPB'),
```
Comment (anselmbradford) [inline careers/_vars-careers.html]: Sorry I sent this wrong, should be `Working at the CFPB`.

File: careers/_vars-careers.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    (path + 'application-process/', 'application-process', 'Job Application Process'),
+    (path + 'students-and-graduates/', 'students-and-graduates', 'Students & Graduates'),
```
Comment (anselmbradford) [inline careers/_vars-careers.html]: Confirm that it should be `Students & Graduates` and not `Students and Graduates` like the page title.

File: careers/students-and-graduates/index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
+{%- endblock %}
+
+{% block content_main %}
+
+    <section class="media block block__flush-top block__sub">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: This doesn't look like it needs `media`. Also, consider stacking the classes.

File: careers/students-and-graduates/index.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+            </p>
+    </section>
+
+    <section class="students-and-graduates">
+
+        <section class="media
+                        block
+                        block__padded-top
+                        block__border-top
+                        u-mb0">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Change `u-mb0` to `block__flush-bottom`. Here and in the rest of the media sections.

File: careers/students-and-graduates/index.html [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+            <div class="media_image-container media_image-container__wide-margin">
+                <img class="u-hide-on-mobile"
+                     src="../../static/img/gavel-round.png">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Perhaps use an absolute URL instead of relative `/static/img/gavel-round.png` ... here and elsewhere in the file.

File: careers/students-and-graduates/index.html [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+                <p class="short-desc">
+                    The Federal government's flagship leadership development program
+                    at the entry level for advanced degree candidates.
+                    The two-year appointment provides a fast-paced opportunity
+                    to gain experience and develop talents.
+                    Fellows will be challenged with opportunities
+                    to flourish into problem solvers, strategic thinkers, and future leaders.
+                </p>
+                <p class="short-desc">
+                    <a href="http://www.pmf.gov/" class="jump-link jump-link__right">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Should this point to a PMF application page URL?

File: careers/students-and-graduates/index.html [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Job Application Process</h2>
+                <p class="short-desc">
+                    The application can be tricky.
+                    We can help.
+                    Learn how the application process works.
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Should be `Learn how it works.`

File: careers/students-and-graduates/index.html [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+                </p>
+                <p class="short-desc">
+                    <a href="#" class="jump-link jump-link__right">
+                        Learn about job application process
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: This is missing from the copy doc. Confirm copy for this link.

File: careers/students-and-graduates/index.html [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+                    </a>
+                </p>
+            </section>
+        </div>
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Follow us on LinkedIn</h2>
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Replace with:

```html
<h2 class="h3">
    <span class="cf-icon cf-icon-linkedin-square"></span>
    Follow us on LinkedIn
</h2>
```

File: careers/students-and-graduates/index.html [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+                <p class="short-desc">
+                    The CFPB is one of the most searched for agencies
+                    in the federal government.
+                    Connect with us to stay updated on the work
+                    we do and new opportunities to be a part of it.
+                </p>
+                <p class="short-desc">
+                    <a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+                       class="jump-link jump-link__right">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Replace classes with `jump-link jump-link__external-link`.

File: careers/students-and-graduates/index.html [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+                        Go to our LinkedIn
+                    </a>
+                </p>
+            </section>
+            <section class="content-l_col-1-2">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: These two need a vertical bar between them, however, it'll be a new style I'm pretty sure so don't worry about it I'm picking it up in another branch.

File: careers/students-and-graduates/index.html [hunk 12]
```diff
@@ -0,0 +1,221 @@
...
+                    We take pride in our feedback-centered culture
+                    that helps us recruit great people.
+                    If you have any feedback, or any specific questions,
+                    let us know at:
+                </p>
+                <p class="short-desc">
+                    <ul class="list list__icons">
+                        <li class="list_item">
+                            <span class="cf-icon cf-icon-email list_icon"></span>
+                            <span class="h5 list_text">Email:</span>
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Confirm whether `Email:` needs colon. I know /contact-us/ has it, but the mocks don't have it. cc @duelj

File: careers/students-and-graduates/index.html [hunk 13]
```diff
@@ -0,0 +1,221 @@
...
+                            <p><a href="mailto:jobs@consumerfinance.gov"
+                               class="jump-link jump-link__right">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Remove `jump-link__right` class.

File: careers/students-and-graduates/index.html [hunk 14]
```diff
@@ -0,0 +1,221 @@
...
+                        </li>
+                    </ul>
+                </p>
+            </section>
+        </div>
+    </section>
+
+    {# Footer #}
+    <aside>
+        <div class="block u-mb0 u-mt0 block__flush-sides block__bg">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Replace `u-mb0` and `u-mt0` with `block__flush-bottom` and `block__flush-top`.

File: careers/students-and-graduates/index.html [hunk 15]
```diff
@@ -0,0 +1,221 @@
...
+            <div class="">
+                    <!-- TODO: Add Office of Civil Rights url -->
+                    {%- import "related-links.html" as related_links -%}
+                    {{- related_links.render([
+                        [ '/blog/',
+                          'CFPB Blog' ],
+                        [ '/newsroom/',
+                          'Newsroom' ],
+                        [ '/equal-employment-opportunity/',
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: You can remove the URL here till this page exists so it's easier to see broken links. It'll get disable styled via https://github.com/cfpb/cfgov-refresh/pull/678

File: careers/students-and-graduates/index.html [hunk 16]
```diff
@@ -0,0 +1,221 @@
...
+                          'Equal Employment Opportunity' ]
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: This should be `Office of Civil Rights`.

File: src/static/css/pages/careers.less [hunk 17]
```diff
@@ -0,0 +1,3 @@
+.students-and-graduates .media_image {
+    width: 150px
```
Comment (anselmbradford) [inline src/static/css/pages/careers.less]: This should perhaps move to the global media_image pattern overall? Sounds like 150px (and 130px at small sizes) is going to be a pattern across the board.

File: careers/_vars-careers.html (hunk 1 above)
Comment (anselmbradford) [inline careers/_vars-careers.html]: Just for the copy, not the URL or ID (unless we want to update those).

File: careers/students-and-graduates/index.html [hunk 18]
```diff
@@ -0,0 +1,221 @@
...
+
+            <h1>Students and graduates</h1>
+            <p class="h3">
+                The CFPB is committed to creating and sustaining
+                a new generation of leaders.
+                There are many opportunities for you to make an immediate impact.
+            </p>
+    </section>
+
+    <section class="students-and-graduates">
```
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: At some point we may want to establish a pattern for page-specific classes. This is essentially the URL, but I wonder if it should include `careers`?

File: careers/students-and-graduates/index.html (hunk 6 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: That's what I saw in the copy doc. @sarahsimpson08?

File: careers/students-and-graduates/index.html (hunk 16 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: There was a discrepancy between the mockup and the finalized copy. Can
@sarahsimpson08 weigh in one too?

File: careers/students-and-graduates/index.html [hunk 19]
```diff
@@ -0,0 +1,221 @@
...
+                <p class="short-desc">
+                    The CFPB is one of the most searched for agencies
+                    in the federal government.
+                    Connect with us to stay updated on the work
+                    we do and new opportunities to be a part of it.
+                </p>
+                <p class="short-desc">
+                    <a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+                       class="jump-link jump-link__right">
+                        Go to our LinkedIn
```
Comment (Scotchester) [inline careers/students-and-graduates/index.html]: Is this the approved copy? Shouldn't it have "profile" or "page" at the end?

File: careers/students-and-graduates/index.html (hunk 11 above)
Comment (Scotchester) [inline careers/students-and-graduates/index.html]: There's a modifier for this.

File: careers/students-and-graduates/index.html (hunk 6 above)
Comment (Scotchester) [inline careers/students-and-graduates/index.html]: Should be an `__external-link` instead of `__right`, too.

File: src/static/css/pages/careers.less (hunk 17 above)
Comment (anselmbradford) [inline src/static/css/pages/careers.less]: Looking into this further I found that these won't always be 150px. For instance, Working with the CFPB has them at 100x100px. I think this is fine for now, but perhaps in the future we should create two `.media_image__*` classes for the two sizes if that ends up being the only sizes we'll have for media image.

File: careers/students-and-graduates/index.html (hunk 18 above)
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: For working with the CFPB I ended up putting this in the `content_modifiers` block to wrap the whole content area and prefixing it with `careers-`. What do you think about removing this here and adding the following after the css block?

```jinja
{% block content_modifiers -%}
    {{ super() }}
    careers-students-and-graduates
{%- endblock %}
```

File: careers/students-and-graduates/index.html (hunk 19 above)
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: According to @sarahsimpson09, final copy for this is `Follow us on LinkedIn`

File: careers/students-and-graduates/index.html (hunk 12 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Just confirmed with @duelj. We're removing colons everywhere.

File: src/static/css/pages/careers.less (hunk 17 above)
Comment (KimberlyMunoz) [inline src/static/css/pages/careers.less]: This really should be consistent against all the Career pages. We're creating so much code for edge-cases that's going to be a pain in the neck to test.

File: careers/students-and-graduates/index.html (hunk 16 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Confirmed with @duelj that it should remain EEO.

Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: NM. Final word from Sarah! https://github.com/cfpb/cfgov-refresh/pull/686#issuecomment-116754972

File: careers/_vars-careers.html (hunk 2 above)
Comment (schaferjh) [inline careers/_vars-careers.html]: Yes, we're using ampersands in navigation and spelling 'and' out in page titles. @sarahsimpson09

File: careers/_vars-careers.html (hunk 1 above)
Comment (KimberlyMunoz) [inline careers/_vars-careers.html]: Fixed. Thanks!

File: careers/students-and-graduates/index.html (hunk 8 above)
Comment (schaferjh) [inline careers/students-and-graduates/index.html]: Let's move forward with:

Short description: "The application can be tricky. We can help." 
Link copy: "Learn how the application process works"

File: careers/students-and-graduates/index.html (hunk 3 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: :scissors: Removed! Thanks!

File: careers/students-and-graduates/index.html (hunk 7 above)
Comment (schaferjh) [inline careers/students-and-graduates/index.html]: In order to work on the link text, let's move forward with:

Short description: "The application can be tricky. We can help." 
Link copy: "Learn how the application process works"

File: careers/students-and-graduates/index.html (hunk 4 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Good call. Changed.

File: careers/students-and-graduates/index.html (hunk 5 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Changed. Let me know if I missed one.

File: careers/students-and-graduates/index.html (hunk 7 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Yep. Fixed.

File: careers/students-and-graduates/index.html (hunk 8 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Checking with @schaferjh on what this link should be.

File: careers/students-and-graduates/index.html (hunk 9 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Fixed. Although I'll probably replace it with your include in another post-sprint PR.

File: careers/students-and-graduates/index.html (hunk 10 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Fixed.

File: careers/students-and-graduates/index.html (hunk 12 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: And fixed.

File: careers/students-and-graduates/index.html (hunk 13 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Fixed.

File: careers/students-and-graduates/index.html (hunk 14 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Fixed.

File: careers/students-and-graduates/index.html (hunk 18 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Fixed

File: careers/students-and-graduates/index.html (hunk 8 above)
Comment (KimberlyMunoz) [inline careers/students-and-graduates/index.html]: Just added Jess' suggestion.

**Comment by KimberlyMunoz** [comment]
I'm going to wait for one more :+1: before squashing

File: src/static/css/pages/careers.less (hunk 17 above)
Comment (anselmbradford) [inline src/static/css/pages/careers.less]: As I understand it there are two sets of image sizes in the media images:
1. **Circle images**.130x130px up to 800px screen-width, then 150x150px.
2. **Non-circular images**. 100x100px at all screen sizes.

I've implemented `media_image__100` and `media_image__130-to-150` in https://github.com/cfpb/cfgov-refresh/pull/686. I think this PR can go ahead and we can consolidate later.

File: careers/students-and-graduates/index.html (hunk 11 above)
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: `.content-l_col__before-inset-divider` added in https://github.com/cfpb/cfgov-refresh/pull/686. The existing modifier that I'm aware of (`.content-l_col__before-divider` extends the line to the top border. This one needs a gap.

File: careers/students-and-graduates/index.html (hunk 18 above)
Comment (anselmbradford) [inline careers/students-and-graduates/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/8ae2c7950368992324794c7aa89771476ae9dabd and https://github.com/cfpb/cfgov-refresh/commit/4d932d3f1245986ac6ecbdd103eb64f18013ed66

File: src/static/css/pages/careers.less (hunk 17 above)
Comment (anselmbradford) [inline src/static/css/pages/careers.less]: Actually this PR should include the size change.
Can you please change:

```less
.students-and-graduates .media_image {
    width: 150px
}
```

…to…

```less
// TODO: Consolidate site-wide media_image responsive sizes into one class.
.students-and-graduates .media_image {
         width: 130px;
        .respond-to-min(800px, {
            width: 150px;
        });
}
```

**Comment by anselmbradford** [comment]
Update https://github.com/cfpb/cfgov-refresh/pull/680/files#r33507042, regenerate assets, and should be :+1: to squash 'n' merge.
